net/wireguard/files/wireguard.sh

```diff
@@ -125,6 +127,9 @@ proto_wireguard_setup() {
   if [ "${preshared_key}" ]; then
     echo "PresharedKey=${preshared_key}" >> "${wg_cfg}"
   fi
+  if [ "${fwmark}" ]; then
+    echo "FwMark=0x${fwmark}" >> "${wg_cfg}"
```

Comment:
Likewise here, fwmark should take either a hexadecimal string starting with `0x`, or a decimal string without `0x`. Then, you don't need to automatically prefix the `0x` when generating the config.